### DISCUSSION TITLE
Salut !

### DISCUSSION THREAD

**Discussion by Clement-Fernandes** [Studio-17/Epitech-Subjects#5] [General]
## 👋 Salut!

Nous utilisons maintenant les discussions comme lieu de partage avec d'autres membres de notre communauté. Nous espérons que vous :
  * Posez les questions que vous vous posez :ticket:
  * Partager des idées :bulb:

Si vous rencontrez un problème sur le repo n'hésitez pas à nous écrire, ça nous donnerais de la motivation pour continuer ainsi que d'améliorer ce repo ! :purple_heart:
 et nous répondrons à vos attentes le plus rapidement possible. :muscle:

**Comment by Ximaz** [comment]
Salut Clément ! C'est vraiment cool d'avoir ouvert ce repo, ça me permet d'apprendre le C en plus de prendre de l'avance.
C'est vraiment excitant, d'autant plus que les sites comme Roslyn n'hébergent pas/plus les sujets récents, ce qui semble être le cas ici.
C'est super gentil d'avoir téléchargé les PDF pour nous les fournir. 👍🏽

**Reply by Clement-Fernandes** [reply]
Aucun soucis et bonne chance à toi pour la suite! 🚀

**Comment by Kaiwinta** [comment]
Salut, je voulais savoir si la récupération des tests de la mouli était fait à la main ou si il y avais un scrapper pour ça. Car je compte faire une PR avec les sujets plus récent et comptais automatiser la récupération des tests

**Reply by Clement-Fernandes** [reply]
Salut, les tests datent d'un moment et pour le coup ils ont été écrit à la main. Le but du repo est d'être à jour le plus possible et le seul moyen c'est que les nouveaux étudiants y participent.
Si tu veux rajouter des nouveaux sujets et scrapper les tests hésite pas ! Ping moi quand tu aura ta PR et je te mergerai ça si tout est bon. 🫡

**Reply by Kaiwinta** [reply]
Salut, mes scrappers sont prêt, le seul problème pour le moment c'est la langue.  J'aime bien le fait de garder l'anglais de l'intra et donc d'avoir le même contenu que l'intra mais n'étant pas Auteur du repo je voulais savoir si créer les Readme des modules en anglais est un problème ou pas.

Je peux voir pour traduire sinon, je veux juste un avis sur la marche à suivre.

Example de ce que le changement ferais: [main](https://github.com/Kaiwinta/Epitech-Subjects/tree/main/Semester-1/B-CPE-110) -> [version avec l'intra](https://github.com/Kaiwinta/Epitech-Subjects/tree/feat/Pge2028-first-year/Semester-1/B-CPE-110)

**Reply by Clement-Fernandes** [reply]
Je pense que l'anglais est pas un soucis. La base de ce repo est écrit en anglais dans le but que tout le monde puisse l'utilise que ce soit des étudiants d'Epitech français ou à l'international comme des des gens qui veulent juste découvrir epitech.
